src/components/Header/index.js

```diff
@@ -102,7 +102,7 @@ export const Header = () => {
                             </button>
                             {isLanguageMenuShown &&
                                 <div className="menu">
-                                    <button className="btn_menu">Eng</button>
+                                    <button className="btn_menu">En</button>
```

Comment:
- UI issues that still exist:

1. Fix arrows to be in right position like design (the right arrow is not aligned properly, Also there is a weird shadow in the right side of the list).
2. The categories card has the elements smaller than design you can spot and feel white spaces that is not in the design.